config/300-samplesource.yaml

```diff
@@ -27,6 +27,33 @@ metadata:
   name: samplesources.samples.knative.dev
 spec:
   group: samples.knative.dev
+  versions:
+    - name: v1alpha1
```

Comment:
see the grouping here: 
https://github.com/knative/eventing/blob/master/config/core/resources/pingsource.yaml#L32-L37

Reply:
here is a sample: https://github.com/knative-sandbox/sample-source/blob/f9ba628480ce133bf8ff777ee1d05872567fc6ff/config/300-samplesource.yaml

Reply:
Thanks, do we update the copyright year as well?